### PR TITLE
fix(textlint-formatter): run all tests

### LINKED ISSUE #402: No formatter test will be executed
Hi, I found issues while implementing possible fix for #400 locally.

**What version of textlint are you using?**

v10.0.0

**What file type (Markdown, plain text, etc.) are you using?**

N/A

**What did you do? Please include the actual source code causing the issue.**

Testing `textlint-formatter` by `yarn run test`

**What did you expect to happen?**

Formatters tests, including tests under [`test/formatters/`](https://github.com/textlint/textlint/tree/c0693b79a48ba84620dc57235667f3d6219e174f/packages/textlint-formatter/test/formatters) directory, will be executed.

**What actually happened? Please include the actual, raw output from textlint.**

Only [`textlint-formatter-test.js`](https://github.com/textlint/textlint/blob/c0693b79a48ba84620dc57235667f3d6219e174f/packages/textlint-formatter/test/textlint-formatter-test.js) was executed.

This seems to be due to glob mismatch in the [`package.json`](https://github.com/textlint/textlint/blob/c0693b79a48ba84620dc57235667f3d6219e174f/packages/textlint-formatter/package.json#L19). Solution could be as follows. Could you give me your thoughts?

```diff
diff --git a/packages/textlint-formatter/package.json b/packages/textlint-formatter/package.json
index 4c1114b..ac9de7d 100644
--- a/packages/textlint-formatter/package.json
+++ b/packages/textlint-formatter/package.json
@@ -16,7 +16,7 @@
     "bin/"
   ],
   "scripts": {
-    "test": "mocha test/*.js"
+    "test": "mocha test/**/*.js"
   },
   "directories": {
     "test": "test/"
```

...and you will get 14 failing tests for `stylish` and `table` formatters 😣

### PR DESCRIPTION
**Pull request by 0x6b**
Correct mocha test glob pattern to run all tests. Closes #402.

### CODE REVIEW

File: packages/textlint-formatter/package.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
     "bin/"
   ],
   "scripts": {
-    "test": "mocha test/*.js"
+    "test": "mocha test/**/*.js"
```
Comment (azu) [inline packages/textlint-formatter/package.json]: Can you try --no-color option?

https://github.com/textlint/textlint/issues/402#issuecomment-352808734

Comment (0x6b) [reply packages/textlint-formatter/package.json]: Run test from top directory without `--no-color` have passed successfully.

Comment (0x6b) [reply packages/textlint-formatter/package.json]: Should I add `--no-color` option to the `package.json`, or leave it as it is (at this moment, and fix test cases later)?

Comment (azu) [reply packages/textlint-formatter/package.json]: Please add `--no-color` option.
master branch should be always passed tests in any environment.

**Comment by 0x6b** [comment]
Thank you!

**Pull request merged**